### DISCUSSION TITLE
Adding dark/white mode switcher in the editor

### DISCUSSION THREAD

**Discussion by BekaArabidze98** [GrapesJS/grapesjs#5626] [Q&A]
Hello. Thanks again for a hard work.
I got one question If anyone knows anything.
So Is there a way to add dark/white switcher in the editor?
But I dont want to change editor look, I wanna change content look inside editor.

I think I should have both dark and white design in the database and fetch
whenever an user clicks on button. But I am sure there will be better approach.
Do you guys have any idea? Maybe example or something.

**Comment by artf** [comment]
Not really but there is already an open discussion https://github.com/GrapesJS/grapesjs/discussions/5084

**Reply by BekaArabidze98** [reply]
Thanks
